src/encoded/commands/load_data.py

```diff
@@ -11,6 +11,7 @@
 
 
 EPILOG = __doc__
+PRODUCTION_NAME_PARTS = ('webprod', 'blue', 'green')
```

Comment:
Thinking on this more, shouldn't the instance names be `fourfront-prod-blue` and `fourfront-prod-green` so that we can test with `fourfront-mastertest-blue` and `fourfront-mastertest-green` if we want to? In that case, `blue` and `green` would not be prod parts, just `prod` would be.

I know that's different than what you advertised. Is anyone depending on the names you chose? Is it a lot of work to change at this point?

Reply:
Yeah, it would be a lot of work to change at this point. We should revisit the naming conventions when we're ready for CGAP. I don't think this is a huge deal for 4DN.